piccolo/query/methods/select.py

```diff
@@ -123,19 +123,35 @@ class Count(Selectable):
     """
 
     def __init__(
-        self, column: t.Optional[Column] = None, alias: str = "count"
+        self,
+        column: t.Optional[Column] = None,
+        distinct: t.Optional[t.Sequence[Column]] = None,
+        alias: str = "count",
     ):
+        if distinct and column:
+            raise ValueError("Only specify `column` or `distinct`")
+
         self.column = column
+        self.distinct = distinct
         self._alias = alias
 
     def get_select_string(
         self, engine_type: str, with_alias: bool = True
     ) -> str:
-        if self.column is None:
-            column_name = "*"
+        expression: str
+
+        if self.distinct:
```

Comment:
Just a question. If user pass multiple columns to `distinct` in Sqlite like this
```python
Band.count(distinct=[Band.popularity, Band.name]).run_sync()
```
Sqlite will raise `sqlite3.OperationalError: row value misused`
If we make change like this (using concatenation operator)
```python
if self.distinct:
    if engine_type == "sqlite":
        column_names = " || ".join(
            i._meta.get_full_name(with_alias=False)
            for i in self.distinct
        )
    else:
        column_names = ", ".join(
            i._meta.get_full_name(with_alias=False) for i in self.distinct
        )
    expression = f"DISTINCT ({column_names})"
```
error will be prevented and everything is fine. Can you look at that. Thanks

Reply:
Interesting - good idea. 👍